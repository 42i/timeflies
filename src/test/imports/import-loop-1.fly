import import-loop-1a.fly

