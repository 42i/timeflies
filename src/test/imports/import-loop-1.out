imports/import-loop-1c.fly:3: WARNING : file imports/import-loop-1.fly already processed
imports/import-loop-1b.fly:10: import trace 
imports/import-loop-1a.fly:5: import trace 
imports/import-loop-1.fly:1: import trace 
Bill of materials:
File: imports/import-loop-1.fly
File: imports/import-loop-1a.fly
File: imports/import-loop-1b.fly
File: imports/import-loop-1c.fly
